source/parser/riscv-parser.cpp

```diff
@@ -132,7 +138,8 @@ RiscvParser::parse(const std::string& text, ParserMode parserMode) {
 
   MemoryAllocator allocator(
       {MemorySectionDefinition("text", 1),
-       MemorySectionDefinition("data", _architecture.getWordSize())});
+       MemorySectionDefinition(
+           "data", _architecture.getWordSize() / _architecture.getByteSize())});
```

Comment:
if you make `_architecture.getWordSize() / _architecture.getByteSize())` a separate variable people can see what you're calculating (if you give it a good name) and it will fit on one line